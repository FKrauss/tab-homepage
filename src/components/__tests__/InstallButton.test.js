/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('InstallButton', () => {
  it('renders without error', () => {
    const InstallButton = require('../InstallButton').default
    shallow(<InstallButton />)
  })

  it('matches expected snapshot', function() {
    const InstallButton = require('../InstallButton').default
    const wrapper = shallow(<InstallButton />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
