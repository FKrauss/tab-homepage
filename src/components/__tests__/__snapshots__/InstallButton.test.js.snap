// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstallButton matches expected snapshot 1`] = `
<WithStyles(Button)
  color="primary"
  onClick={[Function]}
  variant="raised"
>
  Add to Chrome
</WithStyles(Button)>
`;
